.github/actions/test-pylint/action.yml

```diff
@@ -28,7 +32,11 @@ runs:
       run: |
         cd pycode/memilio-${{ inputs.package }}
         mkdir -p build_pylint
-        python setup.py pylint
+        if [ -f tools/run_pylint.py ]; then
+          python tools/run_pylint.py
+        else
+          python -m pylint --rcfile=../pylintrc --load-plugins=pylint_json2html --output-format=jsonextended memilio/ > build_pylint/pylint_extended.json || true
+        fi
```

Comment:
A few things here:
- Unless I missed something, the run_pylint.py scripts run the exact same command as the else case here, right?
- ... except for the `|| true` in the else case, which seems to hide a non-0 return values. Why?
- The run_pylint scripts do not seem to differ, apart from the comment at the top. Maybe we move the script to pycode/, so we can run it manually and use it in the CI here?